.packit.yaml

```diff
@@ -32,11 +32,19 @@ jobs:
     targets:
       - fedora-all
     branch: 'main'
+  - job: copr_build
+    trigger: commit
+    targets:
+      - fedora-all
+    branch: 'main'
+    actions:
+      get-current-version:
+        - bash -c 'rpmspec -q --queryformat "%{VERSION}\n" dnf5.spec | head -n1'
   - job: copr_build
     trigger: pull_request
     targets:
       - fedora-all
-    branch: 'create-pull-request/patch'
+    branch: 'dnf5-5.2.0.0'
```

Comment:
Is this change intentional?

Reply:
yes and no. I was using this PR to test how to use packit to build for different target branches. The issue with the release PR is that it should react on a source branch, not a target branch.

More here: https://github.com/packit/packit/issues/2249

Reply:
This PR could be closed, actually. It was more like a test then a proper release.

Reply:
If you don't need this PR, then close it.